components/Courses/letter-card.tsx

```
@@ -0,0 +1,106 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import { Card } from "@/components/ui/card";
+import type { Course } from "@/types/course";
+import { motion } from "framer-motion";
+import { Pause, Play } from "lucide-react";
+import Link from "next/link";
```

ESLint

[error] 'Link' is defined but never used.

(@typescript-eslint/no-unused-vars)